### PR TITLE
feat: web3 profile feed

### PR DESCRIPTION
**Pull request by Randolph314**
## Description

Closes [#792](https://mask.atlassian.net/browse/MF-792)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/extension/popups/pages/Personas/components/PersonaHeader/UI.tsx [hunk 1]
```diff
@@ -75,21 +75,17 @@ export const PersonaHeaderUI = memo<PersonaHeaderUIProps>(
...
                         <Typography className={classes.identifier}>
                             {formatPersonaFingerprint(fingerprint ?? '', 4)}
                             <CopyIconButton text={fingerprint} className={classes.icon} />
                         </Typography>
                     </div>
-                    <ArrowDropIcon
+                    <ArrowDrop
                         className={classes.arrow}
-                        style={{ transform: isSelectPersonaPage ? 'rotate(-180deg)' : undefined }}
+                        style={{ transform: isSelectPersonaPage ? 'rotate(-180deg)' : undefined, fill: '#767F8D' }}
```
Comment (UncleBill) [inline packages/mask/src/extension/popups/pages/Personas/components/PersonaHeader/UI.tsx]:
```suggestion
                        color='#767F8D'
                        style={{ transform: isSelectPersonaPage ? 'rotate(-180deg)' : undefined }}
```

File: packages/shared/src/UI/components/CollectionDetailCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+        const { classes } = useStyles()
+
+        const icons = relatedURLs?.map((url) => {
+            if (url.includes('etherscan.io')) {
+                return (
+                    <Link href={url} target="_blank" marginRight="12px">
+                        <img
+                            className={classes.linkLogo}
+                            src={new URL('./assets/etherscan.png', import.meta.url).toString()}
+                        />
```
Comment (UncleBill) [inline packages/shared/src/UI/components/CollectionDetailCard/index.tsx]: Ask @xiazhiming to add one to Figma.

File: packages/mask/src/plugins/NextID/components/NextIdPage.tsx [hunk 3]
```diff
@@ -135,8 +144,16 @@ export function NextIdPage({ persona }: NextIdPageProps) {
     const personaActionButton = useMemo(() => {
         if (!personaConnectStatus.action) return null
         const button = personaConnectStatus.hasPersona ? t.connect_persona() : t.create_persona()
+        setDescription(personaConnectStatus.hasPersona ? '' : t.create_persona_intro())
+        const icon = personaConnectStatus.hasPersona ? (
+            <Connect sx={{ marginRight: '8px' }} />
+        ) : (
+            <Identity sx={{ width: 18, height: 18, marginRight: '8px' }} />
```
Comment (UncleBill) [inline packages/mask/src/plugins/NextID/components/NextIdPage.tsx]:
```suggestion
            <Identity size={18} sx={{ marginRight: '8px' }} />
```

File: packages/shared/src/UI/components/ConcealableTabs/index.tsx [hunk 4]
```diff
@@ -161,52 +204,148 @@ export function ConcealableTabs<T extends number | string>({
...
+                        onClose={() => setAnchorEl(null)}>
+                        {uniqBy(addressList ?? [], (x) => x.address.toLowerCase()).map((x) => {
+                            return (
+                                <MenuItem key={x.address} value={x.address} onClick={() => onSelect(x)}>
+                                    <div className={classes.menuItem}>
+                                        <div className={classes.addressItem}>
+                                            {x?.type === SocialAddressType.KV ||
+                                            x?.type === SocialAddressType.ADDRESS ? (
+                                                <ReversedAddress
+                                                    address={x.address}
```
Comment (UncleBill) [inline packages/shared/src/UI/components/ConcealableTabs/index.tsx]: Why add business-related into this component? It should be a business free component.

I notice the above part can be separated.

image

File: packages/shared/src/UI/components/CollectionDetailCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+                                {contribution.formatedAmount} {contribution.symbol}
+                            </Typography>
+                            <div className={classes.dayBox}>
+                                {differenceInCalendarDays(new Date(), new Date(Number(contribution.timeStamp) * 1000))}{' '}
+                                {t.days()}{' '}
+                                {differenceInCalendarHours(
+                                    new Date(),
+                                    new Date(Number(contribution.timeStamp) * 1000),
+                                ) % 24}{' '}
+                                {t.hours()} {t.ago()}
```
Comment (UncleBill) [inline packages/shared/src/UI/components/CollectionDetailCard/index.tsx]:
```suggestion
                                {differenceInCalendarDays(Date.now(), Number(contribution.timeStamp) * 1000)} {t.days()}{' '}
                                {differenceInCalendarHours(Date.now(), Number(contribution.timeStamp) * 1000) % 24}{' '}
                                {t.hours()} {t.ago()}

```

BTW, should we consider singular?

File: packages/shared/src/UI/components/ConcealableTabs/index.tsx [hunk 6]
```diff
@@ -161,52 +204,148 @@ export function ConcealableTabs<T extends number | string>({
...
+                        <Link
+                            className={classes.link}
+                            href={
+                                selectedAddress
+                                    ? explorerResolver.addressLink(ChainId.Mainnet, selectedAddress?.address) ?? ''
+                                    : ''
+                            }
+                            target="_blank"
+                            rel="noopener noreferrer">
+                            <LinkOut color="#767f8d" sx={{ fontSize: '20px' }} />
```
Comment (UncleBill) [inline packages/shared/src/UI/components/ConcealableTabs/index.tsx]:
```suggestion
                            <LinkOut color="#767f8d" size={20} />
```

File: packages/shared/src/UI/components/ConcealableTabs/index.tsx (hunk 4 above)
Comment (UncleBill) [reply packages/shared/src/UI/components/ConcealableTabs/index.tsx]:
```ts
import { MaskTabList } from '@masknet/theme'
```

You might use `MaskTabList` instead, which is also used in trending view.

File: packages/mask/shared-ui/locales/en-US.json [hunk 7]
```diff
@@ -114,6 +114,7 @@
     "copied": "Copied",
     "daily": "Daily",
     "dashboard_no_collectible_found": "No collectible found.",
+    "no_nft_found": "No NFT at the current address",
```
Comment (guanbinrui) [inline packages/mask/shared-ui/locales/en-US.json]: Use a better description.

File: packages/mask/src/plugin-infra/register.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+// This file is a JavaScript file because it's reference to the plugins should not be counted as a project reference.
```
Comment (guanbinrui) [inline packages/mask/src/plugin-infra/register.js]: ?

**Review by UncleBill — DISMISSED** [review]

**Review by nuanyang233 — APPROVED** [review]

**Review by UncleBill — APPROVED** [review]

**Pull request merged**